core/base/src/TApplication.cxx

```diff
@@ -1073,6 +1085,63 @@ void TApplication::Help(const char *line)
+         Printf("   CTRL+A              : move cursor to beginning of line");
+         Printf("   CTRL+Arrow Left     : jump to previous word *");
+         Printf("   CTRL+Arrow Right    : jump to next word *");
+         Printf("   CTRL+B              : move cursor left");
+         Printf("   CTRL+C              : send kSigInt interrupt signal");
+         Printf("   CTRL+D              : remove previous character (or exit ROOT if empty line)");
+         Printf("   CTRL+Del            : remove next word *");
+         Printf("   CTRL+E              : move cursor to end of line");
+         Printf("   CTRL+F              : move cursor right");
+         Printf("   CTRL+G              : move cursor right");
+         Printf("   CTRL+H              : remove previous character");
+         Printf("   CTRL+I              : autocomplete command or print suggestions");
+         Printf("   CTRL+J              : execute command");
+         Printf("   CTRL+K              : remove all characters between cursor and end of line");
+         Printf("   CTRL+L              : clear screen");
+         Printf("   CTRL+M              : execute command");
+         Printf("   CTRL+N              : navigate downwards in command history");
+         Printf("   CTRL+O              : select command of the string-based history search (also Enter)");
+         Printf("   CTRL+P              : navigate upwards in command history");
+         Printf("   CTRL+Q              : ");
+         Printf("   CTRL+R              : reverse string-based history search");
+         Printf("   CTRL+T              : clear screen");
+         Printf("   CTRL+U              : remove all characters between cursor and start of line");
+         Printf("   CTRL+W              : delete previous word");
+         Printf("   CTRL+Y              : paste clipboard content (also CTRL+Shift+V)");
+         Printf("   CTRL+Z              : send kSigStop pause job signal");
+         Printf("   Del                 : remove previous character");
+         Printf("   End                 : move cursor to end of line");
+         Printf("   Enter               : execute command");
+         Printf("   Esc, B              : jump to previous word");
+         Printf("   Esc, Backspace      : remove previous word");
+         Printf("   Esc, C              : character to upper and jump to next word");
+         Printf("   Esc, CTRL+H         : remove previous word");
+         Printf("   Esc, D              : remove next word");
+         Printf("   Esc, Del            : remove previous word");
+         Printf("   Esc, F              : jump to next word");
+         Printf("   Esc, L              : word to lower and jump to it")
+         Printf("   Esc, U              : word to upper and jump to it")
+         Printf("   Esc, 0x7F           : remove previous word");
+         Printf("   0x7F                : remove previous character");
```

Comment:
@couet These two codes are supposed to work on Mac. Could you confirm?

Reply:
Should I also rename them to the "key name" maybe? Is it "Del" or "Backspace" or something like that ?